docker-compose.yml

```diff
@@ -117,8 +117,11 @@ services:
     ports:
       - "8102:8080"
     command: start
+  tsjs-did-sirius-id-driver:
+    image: proximax/tsjs-did-sirius-id-driver:latest
+    ports:
+      - "8105:8080"
   uni-resolver-web:
-    image: universalresolver/uni-resolver-web:latest
+    image: universal-resolver:uni-resolver-web:latest
```

Comment:
Pls revert back to the official image name.
https://hub.docker.com/u/universalresolver